javascript/javascript1/week1/dogage.js

```diff
@@ -0,0 +1,10 @@
+let a = prompt("dogYearOfBirth");
+let b = prompt("dogYearFuture");
```

Comment:
`const` not let 🙂 and good names to allow people after you to understand the code